tf2onnx/onnx_opset/nn.py

```diff
@@ -349,7 +349,7 @@ def any_version(cls, opset, ctx, node, **kwargs):
 
         # prefix with batch dim of [1] to satisfy rank requirements
         input_shape = ctx.get_shape(node.input[0])
-        if len(input_shape) == spatial + 1:
+        if input_shape is not None and len(input_shape) == spatial + 1:
             gb = GraphBuilder(ctx)
```

Comment:
Honestly I'm not sure why this condition is here and if it ever triggers.  The unit tests I tried don't run it.